docs/src/frontend/index.md

```diff
@@ -768,10 +768,11 @@ stdlib.setWalletFallback(stdlib.walletFallback({
   providerEnv: 'TestNet', MyAlgoConnect }));
 ```
 
-If the key `WalletConnect` is provided, and bound to the export of `@reach-sh/stdlib/ALGO_WalletConnect`, then [WalletConnect](https://walletconnect.com/) is used to connect to the [Algorand Wallet](https://algorandwallet.com/) for signing.
+If the key `WalletConnect` is provided, and bound to the `ALGO_WalletConnect` field of the export of `@reach-sh/stdlib`, then [WalletConnect](https://walletconnect.com/) is used to connect to the [Algorand Wallet](https://algorandwallet.com/) for signing.
 For example, this sets the wallet fallback to be WalletConnect and the Algorand TestNet:
 ```js
-import WalletConnect from '@reach-sh/stdlib/ALGO_WalletConnect';
+import * as reachsdk from '@reach-sh/stdlib';
+const WalletConnect = reachsdk.ALGO_WalletConnect;
```

Comment:
I'd write `import { ALGO_WalletConnect as WalletConnect } from '@reach-sh/stdlib';`

Reply:
[b0b8b5d](https://github.com/reach-sh/reach-lang/pull/664/commits/b0b8b5d7c8b6ac33868b17c5f5503189e1ea9701) addresses this.